### DISCUSSION TITLE
List of 3rd party libraries included in DNN?

### DISCUSSION THREAD

**Discussion by rezagarrow** [dnnsoftware/Dnn.Platform#4389] [Q&A] [answered]
Hi, I'm looking for a list of all 3rd party libraries used by DNN.  It keeps coming up with clients during their due diligence.
I poked around past discussions and on the documentation site of DNNSoftware.com, but nothing readily turned up.  Does anyone know if/where such a list is maintained? 

Thanks and apologies if I missed something obvious.

**Comment by mitchelsellers** [comment, marked as answer]
The best location is the LIcenses folder contained within any DNN Platform installation: https://github.com/dnnsoftware/Dnn.Platform/tree/develop/DNN%20Platform/Website/Licenses

**Reply by rezagarrow** [reply]
Thanks Mitch!  This is just what the doctor ordered.

**Reply by mitchelsellers** [reply]
Let us know if you need anything else.  That documentation is the "minimum requirement" for inclusion, and we have re-worked that a bit from what it used to be.

**Reply by rezagarrow** [reply]
Will do Mitch.  Happy to share what we pull together as we go in deeper under the hood as well.
